__tests__/app.test.js

```diff
@@ -352,3 +354,38 @@ describe("/api/reviews", () => {
     return request(app).get("/api/reveiws").expect(404);
   });
 });
+
+describe("/api/comments", () => {
+  describe("DELETE /api/comments/:comment_id", () => {
+    it("DELETE 204 - responds with no content and deletes the given comment", () => {
+      return request(app)
+        .delete("/api/comments/1")
+        .expect(204)
+        .then(({ body }) => {
+          expect(body).toEqual({});
+        })
+        .then(() => {
+          return db.query(`SELECT * FROM comments`);
```

Comment:
I would change this to a query that looks for the comment with the id of 1, just in case for some bizarre reason the comment with id 1 isn't the first one in the database